doc/docs-revamp/dev-onboarding-quick-setup.md

```diff
@@ -0,0 +1,375 @@
+
+#### Cabal
+
+Cabal is a system for building and packaging Haskell libraries and programs. It works well with Stack and is used to manage project dependencies and build configurations. Cabal is installed automatically when you install Stack. 
+
+To verify your Cabal installation, run `cabal –version`. 
+
+### Git
+
+Make sure you have Git installed and configured. See [https://git-scm.com/](https://git-scm.com/). 
+
+### IDE
+
+An IDE provides a comprehensive environment for writing, testing, and debugging your Haskell code. Visual Studio Code (VS Code) is a widely used code editor that supports the Haskell extension. See [https://code.visualstudio.com/](https://code.visualstudio.com/). 
+
+## Testing your environment setup
+
+To verify that your environment is set up correctly and functioning, create and run a simple Haskell program. 
+
+The following simple program fetches and displays the current system time and date. This example involves importing and using libraries to handle dates and times. 
+
+### To create your test program
+
+1. Create a new file named `Main.hs`. 
+2. Add the following code to the file: 
+
+```haskell
+import Data.Time
+main :: IO ()
+main = do
+    currentTime <- getCurrentTime
+    putStrLn $ "Current system time is: " ++ show currentTime
+```
+
+3. Open a terminal in VS Code. 
+4. Navigate to the directory containing `Main.hs`. 
+5. Run the command `stack runghc Main.hs`.
+6. Verify that the output is correct. 
+
+## Core concepts of Plutus smart contracts
+Before we start building our vesting smart contract, let's go over the following core concepts that underpin Plutus smart contracts.
+
+### Datum
+In a vesting contract, the datum represents the state and conditions of the vesting agreement. It includes information such as the beneficiary's public key hash and the deadline for the vesting period. The datum is attached to the UTxO that holds the vested funds and is used by the contract's validator script to enforce the vesting conditions. It ensures that the funds can only be released to the beneficiary after the specified deadline has passed.
+
+### Redeemer
+A redeemer is a piece of data that is provided by the spending transaction. It is used by the validator script to make decisions during the validation process. The redeemer is used to pass dynamic information to the validator script that may not be known at the time the UTXO is created but is only known at the time the UTXO is being spent. In the context of a vesting contract, the redeemer is typically not used, since the contract's logic does not need any additional input from the transaction. Instead, the validation of the transaction is based solely on the datum (vesting conditions) and the script context, such as checking if the deadline has passed and if the transaction is signed by the beneficiary. However, if required, a redeemer could be used to provide additional arguments to the validator script.
```

Comment:
> In the context of a vesting contract, the redeemer is typically not used

The fact that this example does not need a redeemer suggests that it is not a very interesting example.

Reply:
I want to use a simple and straightforward example that is easy to understand from the perspective of someone who has never used Plutus. This is an example from the Plutus Pioneer Program. We can consider other examples if they would more effectively serve the purpose of this onboarding guide.